src/pages/gamesPage/gamesPage.test.tsx

```diff
@@ -492,7 +491,7 @@ describe('<GamesPage />', () => {
 
         act(() => editButton.click())
 
-        expect(wrapper.getByText('Update Game')).toBeTruthy()
+        expect(wrapper.getAllByText('Update Game').length).toEqual(2)
```

Comment:
This change is because the game update form now has "Update Game" as both its title and button text.